tests/BuilderTest.php

```diff
@@ -53,31 +53,31 @@ public function testBuild(): void
         $classMetadata = $this->builder->build(\get_class($c));
 
         $props = $classMetadata->getProperties();
-        $this->assertCount(1, $props, 'Number of properties should match');
+        static::assertCount(1, $props, 'Number of properties should match');
```

Comment:
please revert these changes. i know they are static methods but the convention with phpunit has been to use `$this->`. i prefer to keep it that way to be consisten with other liip git repositories.